deepinv/physics/generator/downsampling.py

```diff
@@ -21,10 +21,13 @@ class DownsamplingGenerator(PhysicsGenerator):
     >>> factor = ds['factor']
 
     .. note::
-        Each batch element has the same downsampling factor and filter, but these can vary from batch to batch.
+        If batch size = 1, a random filter and factor is sampled in (filters, factors) at each step.
+        If batch size > 1, a unique factor needs to be sampled for the whole batch, but filters can vary. In this case,
+        it is recommended to set the `psf_size` argument to ensure that all filters in the batch have the same shape.
 
     :param list[str] filters: list of filters to use for downsampling. Default is ["gaussian", "bilinear", "bicubic"].
     :param list[int] factors: list of factors to use for downsampling. Default is [2, 4].
+    :param tuple[int, int] psf_size: size of the point spread function (PSF) to use for the filters. Default is None.
```

Comment:
describe what happens when it's None (i.e. it will use the default size of the filter from the filter functions)